docs/details/debugging/profiler.rst

```diff
@@ -45,12 +45,13 @@ To enable the profiler, set :c:macro:`LV_USE_PROFILER` in ``lv_conf.h`` and conf
         #include <sys/syscall.h>
         #include <sys/types.h>
         #include <time.h>
+        #include <unistd.h>
```

Comment:
stdint.h?

Reply:
The definition of `syscall` is located at `unistd.h`:
https://man7.org/linux/man-pages/man2/syscall.2.html